src/cljs/ataru/virkailija/application/view.cljs

```diff
@@ -722,14 +722,41 @@
       :disabled  @disabled?
       :on-change #(dispatch [:application/toggle-review-state-setting setting-kwd])}]))
 
+(defn- review-state-comment
+  [state-name selected-hakukohde]
+  (fn [state-name selected-hakukohde]
+    (let [review-note     (subscribe [:state-query [:application :notes selected-hakukohde state-name]])
+          review-notes    (subscribe [:state-query [:application :review-notes]])
+          previous-note   (->> @review-notes
+                               (filter #(= (name state-name) (:state-name %)))
```

Comment:
Täs pitäis varmaan olla kans `(= selected-hakukohde (:hakukohde %))`.

Reply:
Jeps, se puuttuu